tests/testing_four_c_simulation.py

```diff
@@ -678,7 +771,8 @@ def create_model(n_steps):
                 f"{initial_run_name}_102_monitor_dbc.csv",
             ),
             n_dof=9,
-            function=function_nbc,
+            time_span=[10 * 0.5, 21 * 0.5],
+            fun_array=[function_nbc, function_nbc, function_nbc],
```

Comment:
With my comment above, we should be able to use exactly the same call as we used to here, i.e., without having to manually specify the time points.